tests/instances.tcl

```diff
@@ -405,7 +410,15 @@ proc run_tests {} {
 
 # Print a message and exists with 0 / 1 according to zero or more failures.
 proc end_tests {} {
-    if {$::failed == 0} {
+    set sentinel_fd_leaks_file "sentinel_fd_leaks"
+    if { [file exists $sentinel_fd_leaks_file] } {
+        set current_dir [pwd]
+        puts [colorstr red "WARNING: sentinel test(s) failed, there are leaked fds in sentinel, 
+        see output file: $current_dir/$sentinel_fd_leaks_file for more details."]
```

Comment:
lets `cat` that file into the stdout, this way we know the output on github actions and such environment (where we only have the standard output).
something like `exec cat $filename` would do

Reply:
Done.